src/coreclr/debug/daccess/dacimpl.h

```diff
@@ -14,14 +14,11 @@
 #define __DACIMPL_H__
 
 #include "gcinterface.dac.h"
-
-#if defined(TARGET_ARM) || defined(FEATURE_CORESYSTEM) // @ARMTODO: STL breaks the build with current VC headers
 //---------------------------------------------------------------------------------------
 // Setting DAC_HASHTABLE tells the DAC to use the hand rolled hashtable for
```

Comment:
IIRC, the hand-rolled DAC hashtable use to be quite a bit slower than STL `unordered_map` path. We have lived with hand-rolled DAC hashtable for years in .NET Core, so the performance of DAC hashtable is probably acceptable.

Reply:
Are you suggesting drop the option to use `unordered_map`?

Reply:
I am not sure. @dotnet/dotnet-diag Do you have an opinion about `DAC_HASHTABLE`? Also, most of changes in this PR are debugger related, so you may want to take a look at those too.

Reply:
@hoyosjs

Reply:
Just curious, why would we keep a hand rolled hashtable if it provides no benefit?

Reply:
@hoyosjs This and the other diag question are all that remain of this PR. Do you or anyone on the diag team have thoughts here?

Reply:
@noahfalk?  I don't why we use DAC_HASHTABLE.

Reply:
The history predates me as well, this is probably ~2005-2007 era code that I've never needed to touch. My take is that I have no evidence DAC_HASHTABLE is causing us trouble so I'd rather leave it as-is until we have a particular reason to change it.